src/coreclr/debug/di/process.cpp

```diff
@@ -180,7 +180,11 @@ STDAPI DLLEXPORT OpenVirtualProcessImpl2(
     IUnknown ** ppInstance,
     CLR_DEBUGGING_PROCESS_FLAGS* pFlagsOut)
 {
+#ifdef TARGET_WINDOWS
```

Comment:
Maybe we can use LoadLibraryExW in Unix as well and delete LoadLibraryW from PAL; since LoadLibraryW calls LoadLibraryExW in PAL, and other few usages don't seem special.

Reply:
Yep. That is the plan. This function in the PAL is about to get an overhaul. We needed to port this to .NET 8 so this PR was the narrow fix. The broader .NET 9 fix will be coming soon.

Reply:
See https://github.com/dotnet/runtime/pull/91693